apps/core/components/footer/footer-menus/web-page-footer-menu.tsx

```diff
@@ -0,0 +1,23 @@
+import { AvailableWebPages, getWebPages } from '~/client/queries/get-web-pages';
+
+import { BaseFooterMenu } from '../footer-menus';
+
+const filterActivePages = (availableStorePages: AvailableWebPages) => {
+  return availableStorePages
+    .filter((page) => page.isVisibleInNavigation)
```

Comment:
Can we do this filter on the API instead?

Reply:
Great callout, TIL.